### PR TITLE
[Enhancement] make sure nodiscard Status can work in non-test mode

### PR DESCRIPTION
**Pull request by silverbullet233**
Fix remaining compilation issues and add nodiscard keyword to Status in non-test mode.

be ut has too many compilation issues to fix, and we will deal with them later.

TODO:
* fix compilation issues in unit test 
* remove all nodiscard keyword in all function definitions since we have added it before Status

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [x] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 13 / 65 (20.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/runtime/plan_fragment_executor.cpp|0|2|00.00%|[144, 286]|
|:large_blue_circle:|src/exec/table_function_node.cpp|0|1|00.00%|[228]|
|:large_blue_circle:|src/runtime/export_sink.cpp|0|1|00.00%|[177]|
|:large_blue_circle:|src/exec/pipeline/nljoin/spillable_nljoin_build_operator.cpp|0|2|00.00%|[65, 72]|
|:large_blue_circle:|src/exec/spill/input_stream.cpp|0|1|00.00%|[68]|
|:large_blue_circle:|src/exec/pipeline/sort/local_parallel_merge_sort_source_operator.cpp|0|1|00.00%|[89]|
|:large_blue_circle:|src/runtime/buffer_control_block.h|0|1|00.00%|[112]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/spillable_hash_join_probe_operator.cpp|0|2|00.00%|[226, 231]|
|:large_blue_circle:|src/udf/java/java_native_method.cpp|0|1|00.00%|[88]|
|:large_blue_circle:|src/exec/pipeline/nljoin/nljoin_build_operator.cpp|0|1|00.00%|[53]|
|:large_blue_circle:|src/exec/hash_join_node.cpp|0|1|00.00%|[266]|
|:large_blue_circle:|src/udf/java/java_data_converter.cpp|0|1|00.00%|[293]|
|:large_blue_circle:|src/exec/aggregator.cpp|0|1|00.00%|[617]|
|:large_blue_circle:|src/exec/schema_scan_node.cpp|0|1|00.00%|[279]|
|:large_blue_circle:|src/exec/pipeline/sink/iceberg_table_sink_operator.cpp|0|2|00.00%|[70, 315]|
|:large_blue_circle:|src/exec/spill/spiller.hpp|0|2|00.00%|[206, 207]|
|:large_blue_circle:|src/exprs/function_call_expr.cpp|0|2|00.00%|[113, 116]|
|:large_blue_circle:|src/exec/pipeline/exchange/multi_cast_local_exchange.cpp|0|1|00.00%|[209]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/hash_join_probe_operator.cpp|0|1|00.00%|[119]|
|:large_blue_circle:|src/runtime/profile_report_worker.cpp|0|1|00.00%|[36]|
|:large_blue_circle:|src/exec/exec_node.cpp|0|1|00.00%|[324]|
|:large_blue_circle:|src/exec/pipeline/aggregate/aggregate_distinct_blocking_sink_operator.cpp|0|1|00.00%|[72]|
|:large_blue_circle:|src/io/cache_input_stream.cpp|0|1|00.00%|[239]|
|:large_blue_circle:|src/exec/tablet_scanner.cpp|0|2|00.00%|[113, 115]|
|:large_blue_circle:|src/runtime/runtime_state.cpp|0|1|00.00%|[330]|
|:large_blue_circle:|src/common/status.h|0|2|00.00%|[26, 27]|
|:large_blue_circle:|src/exec/spill/spill_components.cpp|0|3|00.00%|[529, 530, 587]|
|:large_blue_circle:|src/storage/rowset_update_state.cpp|0|1|00.00%|[425]|
|:large_blue_circle:|src/runtime/file_result_writer.cpp|0|1|00.00%|[58]|
|:large_blue_circle:|src/exec/topn_node.cpp|0|1|00.00%|[191]|
|:large_blue_circle:|src/storage/tablet_meta_manager.cpp|0|1|00.00%|[1497]|
|:large_blue_circle:|src/exprs/java_function_call_expr.cpp|0|3|00.00%|[111, 123, 258]|
|:large_blue_circle:|src/runtime/runtime_filter_worker.cpp|0|4|00.00%|[559, 560, 578, 582]|
|:large_blue_circle:|src/storage/tablet_updates.cpp|1|2|50.00%|[4449]|
|:large_blue_circle:|src/udf/java/java_udf.cpp|1|2|50.00%|[447]|
|:large_blue_circle:|src/storage/binlog_builder.cpp|1|2|50.00%|[76]|
|:large_blue_circle:|src/exec/sorting/merge_cascade.cpp|1|2|50.00%|[91]|
|:large_blue_circle:|src/exec/olap_scan_prepare.cpp|2|2|100.00%|[]|
|:large_blue_circle:|src/formats/parquet/group_reader.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/stream/scan/stream_scan_operator.h|1|1|100.00%|[]|
|:large_blue_circle:|src/storage/persistent_index.cpp|2|2|100.00%|[]|
|:large_blue_circle:|src/storage/tablet_meta.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/runtime/chunk_cursor.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/pipeline/table_function_operator.cpp|1|1|100.00%|[]|

File: be/src/udf/java/java_data_converter.cpp [hunk 1]
```diff
@@ -290,7 +290,7 @@ Status JavaDataTypeConverter::convert_to_boxed_array(FunctionContext* ctx, std::
             env->DeleteLocalRef(jval);
         } else {
             int buffers_offset = buffers->size();
-            columns[i]->accept(&vistor);
+            (void)columns[i]->accept(&vistor);
```
Comment (stdpain) [inline be/src/udf/java/java_data_converter.cpp:293]: RETURN_IF_ERROR

File: be/src/udf/java/java_native_method.cpp [hunk 2]
```diff
@@ -85,7 +85,7 @@ jlongArray JavaNativeMethods::getAddrs(JNIEnv* env, jclass clazz, jlong columnAd
     auto jarr = env->NewLongArray(array_size);
     jlong array[array_size];
     GetColumnAddrVistor vistor(array);
-    column->accept(&vistor);
+    (void)column->accept(&vistor);
```
Comment (stdpain) [inline be/src/udf/java/java_native_method.cpp:88]: RETURN_IF_ERROR

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 16 / 93 (17.20%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/runtime/plan_fragment_executor.cpp|0|4|00.00%|[144, 286, 477, 479]|
|:large_blue_circle:|src/exec/table_function_node.cpp|0|1|00.00%|[228]|
|:large_blue_circle:|src/runtime/export_sink.cpp|0|1|00.00%|[177]|
|:large_blue_circle:|src/exec/pipeline/nljoin/spillable_nljoin_build_operator.cpp|0|1|00.00%|[65]|
|:large_blue_circle:|src/runtime/data_stream_sender.cpp|0|1|00.00%|[624]|
|:large_blue_circle:|src/exec/spill/spill_components.h|0|1|00.00%|[176]|
|:large_blue_circle:|src/storage/memtable.cpp|0|1|00.00%|[181]|
|:large_blue_circle:|src/exec/pipeline/sink/file_sink_operator.cpp|0|1|00.00%|[110]|
|:large_blue_circle:|src/exec/spill/input_stream.cpp|0|1|00.00%|[68]|
|:large_blue_circle:|src/exec/pipeline/sort/local_parallel_merge_sort_source_operator.cpp|0|1|00.00%|[89]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/spillable_hash_join_probe_operator.cpp|0|2|00.00%|[226, 231]|
|:large_blue_circle:|src/udf/java/java_native_method.cpp|0|1|00.00%|[88]|
|:large_blue_circle:|src/exec/pipeline/nljoin/nljoin_build_operator.cpp|0|1|00.00%|[53]|
|:large_blue_circle:|src/exec/hash_join_node.cpp|0|1|00.00%|[266]|
|:large_blue_circle:|src/exec/pipeline/sink/table_function_table_sink_operator.cpp|0|2|00.00%|[86, 124]|
|:large_blue_circle:|src/udf/java/java_data_converter.cpp|0|1|00.00%|[293]|
|:large_blue_circle:|src/exec/schema_scan_node.cpp|0|1|00.00%|[279]|
|:large_blue_circle:|src/exec/pipeline/sink/iceberg_table_sink_operator.cpp|0|3|00.00%|[34, 70, 315]|
|:large_blue_circle:|src/exec/spill/spiller.hpp|0|3|00.00%|[205, 206, 247]|
|:large_blue_circle:|src/exprs/function_call_expr.cpp|0|2|00.00%|[113, 116]|
|:large_blue_circle:|src/exec/pipeline/exchange/multi_cast_local_exchange.cpp|0|1|00.00%|[209]|
|:large_blue_circle:|src/udf/java/java_udf.cpp|0|2|00.00%|[447, 462]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/hash_join_probe_operator.cpp|0|1|00.00%|[119]|
|:large_blue_circle:|src/runtime/profile_report_worker.cpp|0|2|00.00%|[36, 58]|
|:large_blue_circle:|src/exec/exec_node.cpp|0|1|00.00%|[324]|
|:large_blue_circle:|src/exec/pipeline/aggregate/aggregate_distinct_blocking_sink_operator.cpp|0|1|00.00%|[72]|
|:large_blue_circle:|src/exec/pipeline/sink/hive_table_sink_operator.cpp|0|6|00.00%|[35, 67, 104, 106, 131, 133]|
|:large_blue_circle:|src/exec/tablet_scanner.cpp|0|2|00.00%|[113, 115]|
|:large_blue_circle:|src/runtime/runtime_state.cpp|0|1|00.00%|[330]|
|:large_blue_circle:|src/exec/spill/spill_components.cpp|0|5|00.00%|[154, 529, 530, 581, 587]|
|:large_blue_circle:|src/runtime/result_sink.cpp|0|1|00.00%|[159]|
|:large_blue_circle:|src/storage/rowset_update_state.cpp|0|2|00.00%|[425, 686]|
|:large_blue_circle:|src/exec/pipeline/result_sink_operator.cpp|0|1|00.00%|[83]|
|:large_blue_circle:|src/runtime/file_result_writer.cpp|0|1|00.00%|[58]|
|:large_blue_circle:|src/exec/topn_node.cpp|0|1|00.00%|[191]|
|:large_blue_circle:|src/storage/tablet_meta_manager.cpp|0|1|00.00%|[1466]|
|:large_blue_circle:|src/exprs/java_function_call_expr.cpp|0|3|00.00%|[111, 123, 258]|
|:large_blue_circle:|src/storage/tablet_meta.cpp|0|1|00.00%|[185]|
|:large_blue_circle:|src/exec/analytor.cpp|0|2|00.00%|[366, 368]|
|:large_blue_circle:|src/runtime/runtime_filter_worker.cpp|0|3|00.00%|[559, 578, 582]|
|:large_blue_circle:|src/formats/parquet/group_reader.cpp|1|4|25.00%|[96, 677, 685]|
|:large_blue_circle:|src/storage/tablet_updates.cpp|1|2|50.00%|[4449]|
|:large_blue_circle:|src/exec/aggregator.cpp|1|2|50.00%|[615]|
|:large_blue_circle:|src/storage/binlog_builder.cpp|1|2|50.00%|[76]|
|:large_blue_circle:|src/storage/persistent_index.cpp|2|4|50.00%|[3284, 4767]|
|:large_blue_circle:|src/exec/sorting/merge_cascade.cpp|1|2|50.00%|[91]|
|:large_blue_circle:|src/exec/olap_scan_prepare.cpp|2|2|100.00%|[]|
|:large_blue_circle:|src/exec/pipeline/scan/connector_scan_operator.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/io/cache_input_stream.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/runtime/buffer_control_block.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/stream/scan/stream_scan_operator.h|1|1|100.00%|[]|
|:large_blue_circle:|src/storage/lake/update_manager.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/runtime/chunk_cursor.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/pipeline/table_function_operator.cpp|1|1|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 16 / 96 (16.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/runtime/plan_fragment_executor.cpp|0|4|00.00%|[144, 286, 477, 479]|
|:large_blue_circle:|src/exec/table_function_node.cpp|0|1|00.00%|[228]|
|:large_blue_circle:|src/runtime/export_sink.cpp|0|1|00.00%|[177]|
|:large_blue_circle:|src/exec/pipeline/nljoin/spillable_nljoin_build_operator.cpp|0|1|00.00%|[65]|
|:large_blue_circle:|src/runtime/data_stream_sender.cpp|0|1|00.00%|[624]|
|:large_blue_circle:|src/exec/spill/spill_components.h|0|1|00.00%|[176]|
|:large_blue_circle:|src/storage/memtable.cpp|0|1|00.00%|[181]|
|:large_blue_circle:|src/exec/pipeline/sink/file_sink_operator.cpp|0|1|00.00%|[110]|
|:large_blue_circle:|src/exec/spill/input_stream.cpp|0|1|00.00%|[68]|
|:large_blue_circle:|src/exec/pipeline/sort/local_parallel_merge_sort_source_operator.cpp|0|1|00.00%|[89]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/spillable_hash_join_probe_operator.cpp|0|2|00.00%|[226, 231]|
|:large_blue_circle:|src/udf/java/java_native_method.cpp|0|4|00.00%|[88, 89, 90, 91]|
|:large_blue_circle:|src/exec/pipeline/nljoin/nljoin_build_operator.cpp|0|1|00.00%|[53]|
|:large_blue_circle:|src/exec/hash_join_node.cpp|0|1|00.00%|[266]|
|:large_blue_circle:|src/exec/pipeline/sink/table_function_table_sink_operator.cpp|0|2|00.00%|[86, 124]|
|:large_blue_circle:|src/udf/java/java_data_converter.cpp|0|1|00.00%|[293]|
|:large_blue_circle:|src/exec/schema_scan_node.cpp|0|1|00.00%|[279]|
|:large_blue_circle:|src/exec/pipeline/sink/iceberg_table_sink_operator.cpp|0|3|00.00%|[34, 70, 315]|
|:large_blue_circle:|src/exec/spill/spiller.hpp|0|3|00.00%|[205, 206, 247]|
|:large_blue_circle:|src/exprs/function_call_expr.cpp|0|2|00.00%|[113, 116]|
|:large_blue_circle:|src/exec/pipeline/exchange/multi_cast_local_exchange.cpp|0|1|00.00%|[209]|
|:large_blue_circle:|src/udf/java/java_udf.cpp|0|2|00.00%|[447, 462]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/hash_join_probe_operator.cpp|0|1|00.00%|[119]|
|:large_blue_circle:|src/runtime/profile_report_worker.cpp|0|2|00.00%|[36, 58]|
|:large_blue_circle:|src/exec/exec_node.cpp|0|1|00.00%|[324]|
|:large_blue_circle:|src/exec/pipeline/aggregate/aggregate_distinct_blocking_sink_operator.cpp|0|1|00.00%|[72]|
|:large_blue_circle:|src/exec/pipeline/sink/hive_table_sink_operator.cpp|0|6|00.00%|[35, 67, 104, 106, 131, 133]|
|:large_blue_circle:|src/exec/tablet_scanner.cpp|0|2|00.00%|[113, 115]|
|:large_blue_circle:|src/runtime/runtime_state.cpp|0|1|00.00%|[330]|
|:large_blue_circle:|src/exec/spill/spill_components.cpp|0|5|00.00%|[154, 529, 530, 581, 587]|
|:large_blue_circle:|src/runtime/result_sink.cpp|0|1|00.00%|[159]|
|:large_blue_circle:|src/storage/rowset_update_state.cpp|0|2|00.00%|[425, 686]|
|:large_blue_circle:|src/exec/pipeline/result_sink_operator.cpp|0|1|00.00%|[83]|
|:large_blue_circle:|src/runtime/file_result_writer.cpp|0|1|00.00%|[58]|
|:large_blue_circle:|src/exec/topn_node.cpp|0|1|00.00%|[191]|
|:large_blue_circle:|src/storage/tablet_meta_manager.cpp|0|1|00.00%|[1466]|
|:large_blue_circle:|src/exprs/java_function_call_expr.cpp|0|3|00.00%|[111, 123, 258]|
|:large_blue_circle:|src/storage/tablet_meta.cpp|0|1|00.00%|[185]|
|:large_blue_circle:|src/exec/analytor.cpp|0|2|00.00%|[366, 368]|
|:large_blue_circle:|src/runtime/runtime_filter_worker.cpp|0|3|00.00%|[559, 578, 582]|
|:large_blue_circle:|src/formats/parquet/group_reader.cpp|1|4|25.00%|[96, 677, 685]|
|:large_blue_circle:|src/storage/tablet_updates.cpp|1|2|50.00%|[4449]|
|:large_blue_circle:|src/exec/aggregator.cpp|1|2|50.00%|[615]|
|:large_blue_circle:|src/storage/binlog_builder.cpp|1|2|50.00%|[76]|
|:large_blue_circle:|src/storage/persistent_index.cpp|2|4|50.00%|[3284, 4767]|
|:large_blue_circle:|src/exec/sorting/merge_cascade.cpp|1|2|50.00%|[91]|
|:large_blue_circle:|src/exec/olap_scan_prepare.cpp|2|2|100.00%|[]|
|:large_blue_circle:|src/exec/pipeline/scan/connector_scan_operator.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/io/cache_input_stream.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/runtime/buffer_control_block.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/stream/scan/stream_scan_operator.h|1|1|100.00%|[]|
|:large_blue_circle:|src/storage/lake/update_manager.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/runtime/chunk_cursor.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/pipeline/table_function_operator.cpp|1|1|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 16 / 96 (16.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/runtime/plan_fragment_executor.cpp|0|4|00.00%|[144, 286, 477, 479]|
|:large_blue_circle:|src/exec/table_function_node.cpp|0|1|00.00%|[228]|
|:large_blue_circle:|src/runtime/export_sink.cpp|0|1|00.00%|[177]|
|:large_blue_circle:|src/exec/pipeline/nljoin/spillable_nljoin_build_operator.cpp|0|1|00.00%|[65]|
|:large_blue_circle:|src/runtime/data_stream_sender.cpp|0|1|00.00%|[624]|
|:large_blue_circle:|src/exec/spill/spill_components.h|0|1|00.00%|[176]|
|:large_blue_circle:|src/storage/memtable.cpp|0|1|00.00%|[181]|
|:large_blue_circle:|src/exec/pipeline/sink/file_sink_operator.cpp|0|1|00.00%|[110]|
|:large_blue_circle:|src/exec/spill/input_stream.cpp|0|1|00.00%|[68]|
|:large_blue_circle:|src/exec/pipeline/sort/local_parallel_merge_sort_source_operator.cpp|0|1|00.00%|[89]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/spillable_hash_join_probe_operator.cpp|0|2|00.00%|[226, 231]|
|:large_blue_circle:|src/udf/java/java_native_method.cpp|0|4|00.00%|[88, 89, 90, 91]|
|:large_blue_circle:|src/exec/pipeline/nljoin/nljoin_build_operator.cpp|0|1|00.00%|[53]|
|:large_blue_circle:|src/exec/hash_join_node.cpp|0|1|00.00%|[266]|
|:large_blue_circle:|src/exec/pipeline/sink/table_function_table_sink_operator.cpp|0|2|00.00%|[86, 124]|
|:large_blue_circle:|src/udf/java/java_data_converter.cpp|0|1|00.00%|[293]|
|:large_blue_circle:|src/exec/schema_scan_node.cpp|0|1|00.00%|[279]|
|:large_blue_circle:|src/exec/pipeline/sink/iceberg_table_sink_operator.cpp|0|3|00.00%|[34, 70, 315]|
|:large_blue_circle:|src/exec/spill/spiller.hpp|0|3|00.00%|[205, 206, 247]|
|:large_blue_circle:|src/exprs/function_call_expr.cpp|0|2|00.00%|[113, 116]|
|:large_blue_circle:|src/exec/pipeline/exchange/multi_cast_local_exchange.cpp|0|1|00.00%|[209]|
|:large_blue_circle:|src/udf/java/java_udf.cpp|0|2|00.00%|[447, 462]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/hash_join_probe_operator.cpp|0|1|00.00%|[119]|
|:large_blue_circle:|src/runtime/profile_report_worker.cpp|0|2|00.00%|[36, 58]|
|:large_blue_circle:|src/exec/exec_node.cpp|0|1|00.00%|[324]|
|:large_blue_circle:|src/exec/pipeline/aggregate/aggregate_distinct_blocking_sink_operator.cpp|0|1|00.00%|[72]|
|:large_blue_circle:|src/exec/pipeline/sink/hive_table_sink_operator.cpp|0|6|00.00%|[35, 67, 104, 106, 131, 133]|
|:large_blue_circle:|src/exec/tablet_scanner.cpp|0|2|00.00%|[113, 115]|
|:large_blue_circle:|src/runtime/runtime_state.cpp|0|1|00.00%|[330]|
|:large_blue_circle:|src/exec/spill/spill_components.cpp|0|5|00.00%|[154, 529, 530, 581, 587]|
|:large_blue_circle:|src/runtime/result_sink.cpp|0|1|00.00%|[159]|
|:large_blue_circle:|src/storage/rowset_update_state.cpp|0|2|00.00%|[425, 686]|
|:large_blue_circle:|src/exec/pipeline/result_sink_operator.cpp|0|1|00.00%|[83]|
|:large_blue_circle:|src/runtime/file_result_writer.cpp|0|1|00.00%|[58]|
|:large_blue_circle:|src/exec/topn_node.cpp|0|1|00.00%|[191]|
|:large_blue_circle:|src/storage/tablet_meta_manager.cpp|0|1|00.00%|[1466]|
|:large_blue_circle:|src/exprs/java_function_call_expr.cpp|0|3|00.00%|[111, 123, 258]|
|:large_blue_circle:|src/storage/tablet_meta.cpp|0|1|00.00%|[185]|
|:large_blue_circle:|src/exec/analytor.cpp|0|2|00.00%|[366, 368]|
|:large_blue_circle:|src/runtime/runtime_filter_worker.cpp|0|3|00.00%|[559, 578, 582]|
|:large_blue_circle:|src/formats/parquet/group_reader.cpp|1|4|25.00%|[96, 677, 685]|
|:large_blue_circle:|src/storage/tablet_updates.cpp|1|2|50.00%|[4449]|
|:large_blue_circle:|src/exec/aggregator.cpp|1|2|50.00%|[615]|
|:large_blue_circle:|src/storage/binlog_builder.cpp|1|2|50.00%|[76]|
|:large_blue_circle:|src/storage/persistent_index.cpp|2|4|50.00%|[3284, 4767]|
|:large_blue_circle:|src/exec/sorting/merge_cascade.cpp|1|2|50.00%|[91]|
|:large_blue_circle:|src/exec/olap_scan_prepare.cpp|2|2|100.00%|[]|
|:large_blue_circle:|src/exec/pipeline/scan/connector_scan_operator.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/io/cache_input_stream.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/runtime/buffer_control_block.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/stream/scan/stream_scan_operator.h|1|1|100.00%|[]|
|:large_blue_circle:|src/storage/lake/update_manager.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/runtime/chunk_cursor.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/pipeline/table_function_operator.cpp|1|1|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 16 / 96 (16.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/runtime/plan_fragment_executor.cpp|0|4|00.00%|[144, 286, 477, 479]|
|:large_blue_circle:|src/exec/table_function_node.cpp|0|1|00.00%|[228]|
|:large_blue_circle:|src/runtime/export_sink.cpp|0|1|00.00%|[177]|
|:large_blue_circle:|src/exec/pipeline/nljoin/spillable_nljoin_build_operator.cpp|0|1|00.00%|[65]|
|:large_blue_circle:|src/runtime/data_stream_sender.cpp|0|1|00.00%|[624]|
|:large_blue_circle:|src/exec/spill/spill_components.h|0|1|00.00%|[176]|
|:large_blue_circle:|src/storage/memtable.cpp|0|1|00.00%|[181]|
|:large_blue_circle:|src/exec/pipeline/sink/file_sink_operator.cpp|0|1|00.00%|[110]|
|:large_blue_circle:|src/exec/spill/input_stream.cpp|0|1|00.00%|[68]|
|:large_blue_circle:|src/exec/pipeline/sort/local_parallel_merge_sort_source_operator.cpp|0|1|00.00%|[89]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/spillable_hash_join_probe_operator.cpp|0|2|00.00%|[226, 231]|
|:large_blue_circle:|src/udf/java/java_native_method.cpp|0|4|00.00%|[88, 89, 90, 91]|
|:large_blue_circle:|src/exec/pipeline/nljoin/nljoin_build_operator.cpp|0|1|00.00%|[53]|
|:large_blue_circle:|src/exec/hash_join_node.cpp|0|1|00.00%|[266]|
|:large_blue_circle:|src/exec/pipeline/sink/table_function_table_sink_operator.cpp|0|2|00.00%|[86, 124]|
|:large_blue_circle:|src/udf/java/java_data_converter.cpp|0|1|00.00%|[293]|
|:large_blue_circle:|src/exec/schema_scan_node.cpp|0|1|00.00%|[279]|
|:large_blue_circle:|src/exec/pipeline/sink/iceberg_table_sink_operator.cpp|0|3|00.00%|[34, 70, 315]|
|:large_blue_circle:|src/exec/spill/spiller.hpp|0|3|00.00%|[205, 206, 247]|
|:large_blue_circle:|src/exprs/function_call_expr.cpp|0|2|00.00%|[113, 116]|
|:large_blue_circle:|src/exec/pipeline/exchange/multi_cast_local_exchange.cpp|0|1|00.00%|[209]|
|:large_blue_circle:|src/udf/java/java_udf.cpp|0|2|00.00%|[447, 462]|
|:large_blue_circle:|src/exec/pipeline/hashjoin/hash_join_probe_operator.cpp|0|1|00.00%|[119]|
|:large_blue_circle:|src/runtime/profile_report_worker.cpp|0|2|00.00%|[36, 58]|
|:large_blue_circle:|src/exec/exec_node.cpp|0|1|00.00%|[324]|
|:large_blue_circle:|src/exec/pipeline/aggregate/aggregate_distinct_blocking_sink_operator.cpp|0|1|00.00%|[72]|
|:large_blue_circle:|src/exec/pipeline/sink/hive_table_sink_operator.cpp|0|6|00.00%|[35, 67, 104, 106, 131, 133]|
|:large_blue_circle:|src/exec/tablet_scanner.cpp|0|2|00.00%|[113, 115]|
|:large_blue_circle:|src/runtime/runtime_state.cpp|0|1|00.00%|[330]|
|:large_blue_circle:|src/exec/spill/spill_components.cpp|0|5|00.00%|[154, 529, 530, 581, 587]|
|:large_blue_circle:|src/runtime/result_sink.cpp|0|1|00.00%|[159]|
|:large_blue_circle:|src/storage/rowset_update_state.cpp|0|2|00.00%|[425, 686]|
|:large_blue_circle:|src/exec/pipeline/result_sink_operator.cpp|0|1|00.00%|[83]|
|:large_blue_circle:|src/runtime/file_result_writer.cpp|0|1|00.00%|[58]|
|:large_blue_circle:|src/exec/topn_node.cpp|0|1|00.00%|[191]|
|:large_blue_circle:|src/storage/tablet_meta_manager.cpp|0|1|00.00%|[1466]|
|:large_blue_circle:|src/exprs/java_function_call_expr.cpp|0|3|00.00%|[111, 123, 258]|
|:large_blue_circle:|src/storage/tablet_meta.cpp|0|1|00.00%|[185]|
|:large_blue_circle:|src/exec/analytor.cpp|0|2|00.00%|[366, 368]|
|:large_blue_circle:|src/runtime/runtime_filter_worker.cpp|0|3|00.00%|[559, 578, 582]|
|:large_blue_circle:|src/formats/parquet/group_reader.cpp|1|4|25.00%|[96, 677, 685]|
|:large_blue_circle:|src/storage/tablet_updates.cpp|1|2|50.00%|[4449]|
|:large_blue_circle:|src/exec/aggregator.cpp|1|2|50.00%|[615]|
|:large_blue_circle:|src/storage/binlog_builder.cpp|1|2|50.00%|[76]|
|:large_blue_circle:|src/storage/persistent_index.cpp|2|4|50.00%|[3284, 4768]|
|:large_blue_circle:|src/exec/sorting/merge_cascade.cpp|1|2|50.00%|[91]|
|:large_blue_circle:|src/exec/olap_scan_prepare.cpp|2|2|100.00%|[]|
|:large_blue_circle:|src/exec/pipeline/scan/connector_scan_operator.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/io/cache_input_stream.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/runtime/buffer_control_block.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/stream/scan/stream_scan_operator.h|1|1|100.00%|[]|
|:large_blue_circle:|src/storage/lake/update_manager.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/runtime/chunk_cursor.cpp|1|1|100.00%|[]|
|:large_blue_circle:|src/exec/pipeline/table_function_operator.cpp|1|1|100.00%|[]|

**Review by satanson — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Pull request merged**